src/main/resources/db/migration/v0.37.0.59__htpdatasample_tables.sql

```diff
@@ -0,0 +1,289 @@
+	NO MINVALUE
+	NO MAXVALUE
+	CACHE 1;
+
+ALTER TABLE biosampleage
+	ADD CONSTRAINT biosampleage_pkey PRIMARY KEY (id);
+
+ALTER TABLE biosamplegenomicinformation
+	ADD CONSTRAINT biosamplegenomicinformation_pkey PRIMARY KEY (id);
+
+ALTER TABLE htpexpressiondatasetsampleannotation
+	ADD CONSTRAINT htpexpressiondatasetsampleannotation_pkey PRIMARY KEY (id);
+
+ALTER TABLE microarraysampledetails
+	ADD CONSTRAINT microarraysampledetails_pkey PRIMARY KEY (id);
+
+ALTER TABLE htpexpressiondatasetsampleannotation_note
+	ADD CONSTRAINT htpdatasample_relatednotes_id_uk UNIQUE (relatednotes_id);
+
+ALTER TABLE htpexpressiondatasetsampleannotation
+	ADD CONSTRAINT htpdatasample_microarraysampledetails_id_uk UNIQUE (microarraysampledetails_id);
+
+ALTER TABLE htpexpressiondatasetsampleannotation
+	ADD CONSTRAINT htpdatasample_htpexpressionsample_id_uk UNIQUE (htpexpressionsample_id);
+
+ALTER TABLE htpexpressiondatasetsampleannotation
+	ADD CONSTRAINT htpdatasample_htpexpressionsampleage_id_uk UNIQUE (htpexpressionsampleage_id);
+
+ALTER TABLE htpexpressiondatasetsampleannotation
+	ADD CONSTRAINT htpdatasameple_genomicinformation_id_uk UNIQUE (genomicinformation_id);
+
+CREATE INDEX biosampleage_stage_index ON biosampleage USING btree (stage_id);
+
+CREATE INDEX htpdatasample_anatomicalsite_htpdatasample_index ON htpexpressiondatasetsampleannotation_anatomicalsite USING btree (htpexpressiondatasetsampleannotation_id);
+
+CREATE INDEX htpdatasample_anatomicalsite_samplelocations_index ON htpexpressiondatasetsampleannotation_anatomicalsite USING btree (htpexpressionsamplelocations_id);
+
+CREATE INDEX htpdatasample_assemblyversions_htpdatasample_index ON htpexpressiondatasetsampleannotation_assemblyversions USING btree (htpexpressiondatasetsampleannotation_id);
+
+CREATE INDEX htpdatasample_externaldatabaseentity_datasetids_index ON htpexpressiondatasetsampleannotation_externaldatabaseentity USING btree (datasetids_id);
+
+CREATE INDEX htpdatasample_externaldatabaseentity_htpdatasample_index ON htpexpressiondatasetsampleannotation_externaldatabaseentity USING btree (htpexpressiondatasetsampleannotation_id);
+
+CREATE INDEX htpdatasample_note_htpdatasample_index ON htpexpressiondatasetsampleannotation_note USING btree (htpexpressiondatasetsampleannotation_id);
+
+CREATE INDEX htpdatasample_note_relatednotes_index ON htpexpressiondatasetsampleannotation_note USING btree (relatednotes_id);
+
+CREATE INDEX htpdatasetsampleannotation_createdby_index ON htpexpressiondatasetsampleannotation USING btree (createdby_id);
+
+CREATE INDEX htpdatasetsampleannotation_dataprovider_index ON htpexpressiondatasetsampleannotation USING btree (dataprovider_id);
+
+CREATE INDEX htpdatasetsampleannotation_htpexpressionsample_index ON htpexpressiondatasetsampleannotation USING btree (htpexpressionsample_id);
+
+CREATE INDEX htpdatasetsampleannotation_updatedby_index ON htpexpressiondatasetsampleannotation USING btree (updatedby_id);
+
+ALTER TABLE biosamplegenomicinformation
```

Comment:
@VarunReddy1111 everyy single one of these FK's needs to have a corresponding index.